com.alizadicomviewer.AlizaMS.metainfo.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
metainfo info file is missing `<content_rating>` and `<releases>`

Reply:
Just a question. Is `<releases>` mandatory? What if upstream doesn't include this tag?